src/frontend/screens/Library/components/InstallModal/SideloadDialog/index.scss

```diff
@@ -1,7 +1,7 @@
 .sideloadGrid {
   display: grid;
   justify-content: space-around;
-  grid-template-columns: 1fr 3fr;
+  grid-template-columns: min-content 1fr;
```

Comment:
sideload dialog was growing in width as the title got longer


https://github.com/user-attachments/assets/ee088ebc-b1c0-4e24-8eb4-f576048ff03a

and now:
![image](https://github.com/user-attachments/assets/7cf37c6d-07c6-443a-8e55-4f40a41fdeb8)